Author: Daniel Leidert <dleidert@debian.org>
Description: There was a conflicting declaration of the
 rpopup_bevent_in_html_code() function in src/rcfile.c and src/rcfile.h.
Bug-Debian: https://bugs.debian.org/747963

--- a/src/plugin_htmlbar/rpopup.h
+++ b/src/plugin_htmlbar/rpopup.h
@@ -20,7 +20,7 @@
 #ifndef __RPOPUP_H_
 #define __RPOPUP_H_
 
-gboolean rpopup_bevent_in_html_code(Tdocument *doc);
+void rpopup_bevent_in_html_code(Tdocument *doc);
 gboolean rpopup_doc_located_tag(Tdocument *doc);
 gboolean rpopup_doc_located_color(Tdocument *doc);
 void rpopup_edit_tag_cb(GtkMenuItem *menuitem,Tdocument *doc);
